e2e/run_e2e.sh

```diff
@@ -18,6 +18,7 @@ fi
 echo "TEST ClusterPermission"
 kubectl config use-context kind-hub
 kubectl apply -f config/samples/rbac.open-cluster-management.io_v1alpha1_clusterpermission.yaml -n cluster1
+kubectl apply -f config/samples/clusterpermission_existing_roles.yaml -n cluster1
```

Comment:
This should be at the bottom. So you can start your own set of tests.

Your sample contains references to namespaces that doesn't exist. Maybe you should copy https://github.com/open-cluster-management-io/cluster-permission/blob/main/config/samples/rbac.open-cluster-management.io_v1alpha1_clusterpermission.yaml and adjust it to match your test case.

Reply:
Thanks Mike that's a good catch! I will make those changes.